base/sparse/sparsematrix.jl

```diff
@@ -2325,7 +2326,7 @@ function getindex(A::SparseMatrixCSC{Tv}, I::AbstractArray) where Tv
         for r in colptrA[col]:(colptrA[col+1]-1)
             @inbounds if rowvalA[r] == row
                 rowB,colB = ind2sub(szB, i)
-                colptrB[colB+1] += 1
+                colptrB[colB+1] += one(Ti)
```

Comment:
This should not be necessary; the `1` will be converted to the correct type when entered into the vector.

Reply:
Changed.